app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class Users::RegistrationsController < Devise::RegistrationsController
+  
+  def create
+    @user = User.new(configure_sign_up_params)  
+    if @user.save
+      render json: @user
+    else
+      render json: @user.errors, status: :unprocessable_entity
+    end
+  end
+
+  respond_to :json
+
+  private
+
+  def configure_sign_up_params
+    params.require(:user).permit(:email, :password, :password_confirmation, :admin)
+  end
+
+  def respond_with(resource, _opts = {})
+    render json: {
+      status: {code: 200, message: 'Logged in successfully'},
```

Comment:
@josephelias-rubyians  Hi, I believe the opts needs to be placed over here with default for 200.
So for error response and success response it will provide appropriate status. 

OR
  you can define separate success and error response.

What do you think?